contracts/staking/PoSValidatorManager.sol

```diff
@@ -104,6 +105,11 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         _initializeEndValidation(validationID);
     }
 
+    function completeEndValidation(uint32 messageIndex) external {
+        Validator memory validator = _completeEndValidation(messageIndex);
+        _unlock(validator.weight, validator.owner);
+    }
```

Comment:
I know this PR doesn't integrate delegation yet, but I'm going to leave a comment on Cam's PR... We're going to need to probably track the validator's initial weight along with its total weight with delegations. As-is, if this were integrated with the delegation PR, this would unlock all the tokens delegated to the validator to the validator.

Reply:
addressed in https://github.com/ava-labs/teleporter/pull/527/commits/1ef6e2d0c8747dbbe98a3985248f1753e9dcb5d7